data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,596 @@
+	thumbnail "outfit/photoelectric lamina"
+	"mass" 24
+	"anchor point" -1
+	"solar collection" 6
+	"solar heat" 1.8
+	description "These concentrating solar panels can be mounted conformally to a ship's hull, efficiently collecting sunlight and turning it into usable energy."
+	description "Photovoltaic Laminae take advantage of special external hardpoints built into most Avgi ships, allowing them to be mounted without taking up any outfit space. However, they also tend to increase the amount of heat absorbed from the local star."
+
+
+outfit "Lantern Fission Core"
+	category "Power"
+	cost 1200000
+	thumbnail "outfit/lantern fission core"
+	"mass" 24
+	"outfit space" -24
+	"energy generation" 3.6
+	"heat generation" 12
+	description "The Lantern is a compact dusty plasma fission reactor designed to fit even on large fighters or interceptors. By suspending uranium dust in a hydrogen plasma, which is in turn contained by a magnetic field, the reactor can achieve incredible core temperatures high enough to make it surprisingly efficient for its size, at the cost of producing significant amounts of waste heat."
+
+
+outfit "Ultracapacitor Cell"
+	category "Power"
+	cost 20400
+	thumbnail "outfit/unknown"
+	"mass" 1
+	"outfit space" -3
+	"energy capacity" 1500
+	"ion protection" -0.01
+	"description" "The internal structure of this deceptively light capacitor cell resembles a fractal of molecule-thin graphene sheets interlocked together. While not the most efficient in their use of outfit space, this capacitor cell is extraordinarily energy dense for its mass, a deliberate design choice from the days of sublight interstellar travel, when every gram counted."
+	"description" "Unfortunately, the internal structure is also quite delicate, and is prone to potential discharge when exposed to ionizing radiation. Thankfully, resilient grounding leads prevents this discharge from becoming catastrophic."
+	
+	
+outfit "Ultracapacitor Bank"
+	category "Power"
+	cost 200000
+	thumbnail "outfit/unknown"
+	"mass" 6
+	"outfit space" -18
+	"energy capacity" 10800
+	"ion protection" -0.03
+	"description" "These fractal capacitors are capable of storing a large amount of energy within superconducting graphene sheets. This capacitor bank chains together six individual cells, but saves space by discarding redundant components and unnecessary structural mass."
+	"description" "Like its component cells, the capacitor bank is vulnerable to ionizing weapons, but is relatively better shielded for its size by virtue of its reduced surface area to volume ratio."
+
+
+# Engines:
+
+outfit "Magnetoplasma Drive"
+	category "Engines"
+	"cost" 30000
+	thumbnail "outfit/magnetoplasma drive"
+	"mass" 12
+	"outfit space" -12
+	"engine capacity" -12
+	"thrust" 9
+	"thrusting energy" 0.6
+	"thrusting heat" 0.3
+	"reverse thrust" 9
+	"reverse thrusting energy" 0.6
+	"reverse thrusting heat" 0.3
+	"turn" 270
+	"turning energy" 0.6
+	"turning heat" 0.3
+	"drag reduction" 0.15
```

Comment:
At what the game shows at 15 drag reduction, which I suspect is this 0.15, a ship like a Remnant Gull loaded with eight Magnetoplasma Drives (filling its engine space) reaches speeds that outmatch Korath engines. A Hauler III loaded up with six reaches 352 speed, which likewise outmatches a relatively balanced Korath engine layout for less heat and less power consumption. They distinctly outperform Remnant engines and Atomic engines, which I believe to be the best comparison; though their turning capability is fairly similar. For a truly extreme example, loading up an Augen takes its top speed from 109 with Gegno engines, past 172 on a Korath engine set designed to mimic the performance of the 38 Magnetoplasma Drives (one each of Stellar+Planetary thrust and reverser, two Planetary steering), past 185 on a Remnant engine set, past 240 on Quarg engines, past 270 on an impractical double Sheragi Fusion Drive loadout, all the way to slightly exceeding Korath afterburners at a total 1146 speed. Obviously this is infeasible, the Augen is not available to the player, but player-accessible ships can still mount up to 23 Magnetoplasma Drives- which rockets them to speeds only matchable by pure-afterburner builds, and then with much less reverse capacity and much less heat production (though similar power consumption). 

(Sorry for all the examples, they're because I got distracted in Omnis rather than an attempt to make an Epic Debate Win.)

If the Avgi specialty is to be speed, I believe this is better represented focusing on their ships themselves rather than in engines that, en masse, are on-par with afterburners.

Reducing the drag reduction may help, but I'm not sure if it'll do the trick fully? A ship like a Star Barge can mount three Magnetoplasma Drives and gets speeds that just about match a set of Hai thrusters that fill the same space currently because it has so much less drag reduction, so obviously the effect is going to be much smaller on smaller ships, but it makes sense for smaller ships to underperform and larger ones to overperfrom. A drag reduction of 0.1, rather than 0.15, makes something like a Star Barge remain fairly fast- on par with Ion thrusters filling the same space- but something like a worldship remains significantly faster than on a Korath engine set... hmm.